_posts/2016-09-30-subtype-typeclasses.md

```diff
@@ -89,13 +95,13 @@ def bar[F[_]: Applicative: Functor]: F[Int] = Applicative[F].pure(10)
 ```
 
 ```scala
-def callBar[F[_]: Monad: Traverse]: F[Int] = bar[F]
-// <console>:19: error: ambiguous implicit values:
-//  both value evidence$2 of type cats.Traverse[F]
-//  and value evidence$1 of type cats.Monad[F]
-//  match expected type cats.Functor[F]
-//        def callBar[F[_]: Monad: Traverse]: F[Int] = bar[F]
-//                                                        ^
+scala> def callBar[F[_]: Monad: Traverse]: F[Int] = bar[F]
```

Comment:
Do we expect the compiled (from `tut`) code to be part of commits? For instance the `tut` file changed after https://github.com/larsrh/typelevel.github.com/pull/1 but the corresponding post does not reflect that (there should be no difference between what's currently in master and what is generated from the `tut` I PR'd).